app/services/users.js

```diff
@@ -26,3 +28,94 @@ exports.existUserMail = async mail => {
     return new Error(`signUpUser Error => ${error.message}`);
   }
 };
+
+exports.getUserByEmailAndPassword = async (mail, newPassword) => {
+  try {
+    let hasCorrectParameters = false;
+    const userData = await userModel.findOne({ where: { mail } });
+
+    if (objIsNotEmpty(userData) && userData.password) {
+      hasCorrectParameters = await checkPassword(newPassword, userData.password);
+    }
+
+    return hasCorrectParameters ? userData : {};
+  } catch (error) {
+    logger.error(`getUserByEmailAndPassword Error => ${error}`);
+    return new Error(`getUserByEmailAndPassword Error => ${error.message}`);
+  }
+};
+
+exports.getUserByAttribute = async condition => {
+  try {
+    const user = await userModel.findOne({ where: { ...condition } });
+
+    return user;
+  } catch (error) {
+    logger.error(`getUserByAttribute Error => ${error}`);
+    return new Error(`getUserByAttribute Error => ${error.message}`);
+  }
+};
+
+exports.getUserById = id => {
```

Comment:
no need to return user. You could to return the findOne and I prefer use promise instance of async await. For example

exports.getUserByAttribute = condition => userModel.findOne({ where: { ...condition } }) .catch(error => { logger.error(' .... '); throw dataBaseError('.....'); })